packages/classic/README.md

```diff
@@ -119,13 +117,11 @@ To ensure version predictability, we use [Semantic Versioning](https://semver.or
 
 This package is part of the following suites:
 
-* [Kendo UI for Angular](http://www.telerik.com/kendo-angular-ui/)
-* [KendoReact](http://www.telerik.com/kendo-react-ui/)
-* [Kendo UI for jQuery](http://www.telerik.com/kendo-ui)
-* [UI for ASP.NET MVC](http://www.telerik.com/aspnet-mvc)
-* [UI for ASP.NET Core](http://www.telerik.com/aspnet-core-ui)
+* [Telerik Kendo UI for jQuery](https://www.telerik.com/kendo-ui)
```

Comment:
We don't need the "Telerik" name in front of Kendo UI.
The full name of all of our brands is "Progress [brand]"
From then on, all the JavaScript UI libraries are Kendo UI and all the .NET - Telerik UI. That is, we don't use "Telerik" in the context of JavaScript.
If in doubt, you can always check our official brand names here: https://www.progress.com/legal/trademarks
We don't need to use the (tm) and (r) unless it's an external property and this is the first product mention.
Same with the long name - we don't need to use "Progress Kendo UI" on our website, just "Kendo UI for jQuery"

Reply:
I understand. Thank for the clarification :)